test/functional/ui/cmdline_spec.lua

```diff
@@ -623,8 +623,7 @@ local function test_cmdline(linegrid)
       { 'unplace', '', '', '' },
     }
 
-    command('set wildmode=full')
-    command('set wildmenu')
+    command('set wildmode=full wildmenu wildoptions=pum')
```

Comment:
Is this just a clean up of the tests to make them more explicit, or is this necessary due to a change of behavior?

Reply:
Includes a behavior change. In this PR ext_popupmenu doesn't (I thought it shouldn't) receive events unless the user sets wildoptions=pum. ext_wildmenu still does regardless of the option.

Reply:
ext_popupmenu also includes wildmenu (regardless of options):

> - `ext_popupmenu`	Externalize |popupmenu-completion| and
>			'wildmenu'. |ui-popupmenu|

Adding a separate `ext_wildmenu` format  was a mistake and this option is now deprecated.

Reply:
Yes but if the user indicates they don't want a popupmenu for the cmdline through wildoptions, why would we still emit the event?

Reply:
Behavior changes LGTM, ok w/ @bfredl ?